documentation/milestones/individual_milestones/search_mvp.md

```diff
@@ -0,0 +1,218 @@
+- Which fields should support filtering and sorting?
+- Do we use an existing search service (e.g. Elasticsearch) or build our own?
+
+### Search Page
+
+The primary page we'll need to deliver for the user interface (UI) is a search page, which should allow users to:
+
+- Enter key words to search the list of funding opportunities
+- Filter search results by *at least* one structured field (e.g. award date, opportunity type, etc.)
+- View a common set of fields for each opportunity returned by the search (e.g. title, award date, summary, etc.)
+- Sort the set of results returned by *at least* one structured field (e.g. award date, title, etc.)
+
+### Opportunity Listing Page
+
+The secondary page we want to deliver within the user interface (UI) is an opportunity listing page, which should:
+
+- Display a consistent list of metadata about each opportunity
+- Use only fields that are available in the existing grants.gov cloud database
+- Link to the corresponding opportunity in the existing grant.gov interface
+
+**Note:** This is a stretch goal for this deliverable and may be substituted with a direct link to the grants.gov page for a given opportunity from the search results.
+
+### Definition of done
+<!-- Required -->
+
+- [ ] Basic requirements:
+  - [ ] Code is deployed to `main` & PROD through our CI/CD pipeline
+  - [ ] Services are live in PROD (may be behind feature flag)
+  - [ ] Translations are live in PROD (if necessary)
+  - [ ] All new services have passed a security review (if necessary)
+  - [ ] All new services have completed a 508 compliance review (if necessary)
+  - [ ] Data needed for metrics is actively being captured in PROD
+  - [ ] Key architectural decisions made about this deliverable are documented publicly
+- [ ] Functional requirements:
+  - [ ] Users can search for opportunities by key word
+  - [ ] Users can filter search results by *at least one* structured field (e.g. award date, opportunity type, etc.)
+  - [ ] Users can sort search results by *at least* one structured field (e.g. award date, title, etc.)
+  - [ ] All of the search features available in the UI are also available via the API, and vice versa
+  - [ ] Search criteria are reflected in the URL so that users can bookmark or share the link to a given set of search results
+  - [ ] Users can access the corresponding grants.gov page for an opportunity they find on simpler.grants.gov via link on the search results or opportunity listing page
+
+### Proposed metrics for measuring goals/value/definition of done
+<!-- Required -->
+
+1. Total number of searches made via the UI
+2. Total number of searches made via the API
+3. Number of results returned per search
+4. Frequency distribution of keyword search terms
+5. Frequency distribution of field-based filters
+6. Click conversion rate from search results to opportunity listing page
+
+### Destination for live updating metrics
+<!-- Required -->
```

Comment:
What do you think of having a metrics dashboard that's manually updated in the wiki for now, until we have a decision on the public measurement dashboard.

Reply:
Yeah I like this idea and I think it meets Lucas' need. We have a [similar call out on other pages](https://github.com/HHS/simpler-grants-gov/blob/main/documentation/milestones/individual_milestones/static_site.md#destination-for-live-updating-metrics)